src/i_video.c

```diff
@@ -304,11 +304,6 @@ static boolean MouseShouldBeGrabbed(void)
    if (!window_focused)
       return false;
    
-   // always grab the mouse when full screen (dont want to 
-   // see the mouse pointer)
-   if (fullscreen)
```

Comment:
`if (fullscreen && !menuactive)` maybe?

I wouldn't want to see the mouse e.g. on the intermission screen.

Reply:
How about this fix? https://github.com/fabiangreffrath/woof/pull/956/commits/4552a9982bad8e73ee9d86f5eb0177dc7bc9e6a1